### ISSUE TITLE
[feature]: Order note at checkout

### ISSUE THREAD

**Issue by plamenh** [magento/pwa-studio#3075]
**Is your feature request related to a problem? Please describe.**
PWA can't select a default country when rendering a directory Form like signup or checkout so Customers cannot see US as default making it frustrating to scroll every-time they fill out forms

**Describe the solution you'd like**
Expose default country in store config and Country object so PWA devs can choose which one to use.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Please let us know what packages this feature is in regards to:**
- <x] `venia-concept`
- [x] `venia-ui`
- [ ] `pwa-buildpack`
- [x] `peregrine`
- [ ] `pwa-devdocs`
- [ ] `upward-js`
- [ ] `upward-spec`
- [ > `create-pwa`

**Acceptance criteria**
* PWA developers must find in schema a default country in storeConfig and Country object
* fields must be covered with tests

**Comment by sirugh** [comment]
@magento export issue to JIRA project PWA as Story

**Comment by supernova-at** [comment]
Hey @plamenh,

Thanks for writing this up. There is no GraphQL support for this. Additionally, as a feature request, we have decided not to prioritize this at this time, either and as such I have closed this issue.

P.S. the JIRA sync bot completely replaced this issue description with some other problem statement. I copy/pasted the original description back from the associated JIRA ticket. Weird.
